### DISCUSSION TITLE
Properties Support for Breadcrumbs?

### DISCUSSION THREAD

**Discussion by bodiequirk** [SkepticMystic/breadcrumbs#499] [Ideas]
I have noticed some new updates to Breadcrumbs recently, but wasn't sure what what added. I would love to start using Obsidian's new Properties function with my Breadcrumbs data, but wasn't sure if it was supported yet or not. Thanks Ross!

**Comment by SkepticMystic** [comment]
Hey Bodie :)
I'm not super familiar with the properties feature. But my understanding is that it's just a pretty wrapper around YAML frontmatter, is that right? I tested quick, and was able to add a Breadcrumbs link in the properties of a note:

![image](https://github.com/SkepticMystic/breadcrumbs/assets/70717676/341eeec5-c125-490a-8cf0-9284433181ac)

The YAML is as follows:
```yaml
---
up: [[dad]]
---
```

Let me know if that works for you, or if you meant something else.

**Comment by bodiequirk** [comment]
Oh, that may be the case! I am used to using Dataview-style:: double colons and I think I tried the double-colon approach inside the YAML and it didn't work, but I haven't tried it in awhile. Right now I just add a makeshift "footer" at the bottom of my notes where I include all the Dataview values including my Parent / child Breadcrumbs assignments.

**Comment by michaelpporter** [comment]
Closing — v4 reads frontmatter Properties natively. Reopen if something specific is missing.
